pantheon.php

```diff
@@ -40,4 +40,7 @@
 		 */
 		define( 'FS_METHOD', 'direct' );
 	}
+    if ( defined( 'WP_ALLOW_MULTISITE' ) && ( ! defined( 'MULTISITE' ) || empty( MULTISITE ))) {
```

Comment:
Silly wp spacing standars:
```suggestion
    if ( defined( 'WP_ALLOW_MULTISITE' ) && ( ! defined( 'MULTISITE' ) || empty( MULTISITE ) ) ) {
```